Assignment2.yaml

```diff
@@ -0,0 +1,402 @@
+swagger: "2.0"
+info:
+  description: "Starwars API."
+  version: "1.0.0"
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/api/"
+tags:
+- name: "Vehicle"
+  description: "Everything about your Vehicle"
```

Comment:
externalDocs is the link to the document.
here: http://swapi.co/documentation#vehicles